apps/server/src/stores/runtimeState.ts

```diff
@@ -521,10 +522,15 @@ export function update(): UpdateResult {
   }
 
   if (runtimeState.blockNow) {
-    const expectedBlockFinish = getExpectedBlockFinish(runtimeState, getCurrentRundown());
+    const expectedBlockFinish = getExpectedBlockFinish(runtimeState, getCurrentRundown()); //TODO: this in pulling in rundown by it self, we are tying not to do that in this file
```

Comment:
question: could we not have saved the block information from the load? this will take the rundown at every tick (32ms) which is not ideal)

how about either:
- we save the block in a private object like we do with `_timer`
- we expand `blockNow` to contain the data we need